test/integration/examples/apiserver_test.go

```diff
@@ -769,7 +779,8 @@ func runPreparedWardleServer(
 		if flunderBanningFeatureGate {
 			args = append(args, "--feature-gates", fmt.Sprintf("wardle:BanFlunder=%v", banFlunder))
 		}
-		wardleCmd := sampleserver.NewCommandStartWardleServer(ctx, wardleOptions)
+		// TODO figure out how to actually make BinaryVersion/EmulationVersion work with Wardle and KAS at the same time when Alpha FG are being set
+		wardleCmd := sampleserver.NewCommandStartWardleServer(ctx, wardleOptions, withUID)
```

Comment:
the parameter name for new one threw me off somewhat .. we are using `withUID` here and in start.go in the definition of the `NewCommandStartWardleServer` method we are using `skipDefaultComponentGlobalsRegistrySet`